releasenotes/notes/fix-security-agent-start-bd677ec89892a959.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix init of security-agent - exit properly when no feature requiring it is activated
```

how about, for completeness:

```suggestion
    Fix init of security-agent - exit properly when no feature requiring it is activated and avoid conflicting with core agent port bindings.
```